keps/sig-node/2043-pod-resource-concrete-assigments/README.md

```diff
@@ -113,8 +113,9 @@ This gRPC service returns information about
 - the devices which kubelet knows about from the device plugins.
 - the kubelet's assignment of devices and cpus with NUMA id to containers.
 The GRPC service obtains this information from the internal state of the kubelet's Device Manager and CPU Manager respectively.
-The GRPC Service exposes two endpoints:
+The GRPC Service exposes these endpoints:
 - `List`, which returns a single PodResourcesResponse, enabling monitor applications to poll for resources allocated to pods and containers on the node.
+- `GetAllocatableResources`, which returns AllocatableResourcesResponse, enabling monitor applications to learn about the the allocatable resources available on the node.
```

Comment:
Is there a reason why we need to Watch pod resources, but not allocatable resources?

Reply:
The rationale is the following: to accomodate the usecases we are aware of, we are proposing the minimal amount of changes, and while there is a clear need to Watch() the pod resources, there is no clear need to Watch() the allocatable resources. We are open to more extensions in the future.
That said, I need to check in parallel if kubelet can deal with resources hotplug, because last update I got was: https://github.com/kubernetes/kubernetes/issues/67500 . So I need to check if this was fixed.

Reply:
It is at least possible to install a new device plugin, which would update the list of resources.

Reply:
good point. This calls for a review of the Watch() part, which was postponed for 1.21 anyway (capacity reasons).